ui/src/components/References/GroupReference.tsx

```diff
@@ -101,7 +101,7 @@ export default function GroupReference({
           </span>
         ) : (
           <>
-            {gang.invite && status !== 'loading' ? (
+            {gang.invite && button.text !== 'Go' && status !== 'loading' ? (
```

Comment:
might be safer to check if the group doesn't exist

Reply:
Good idea.